sw/device/lib/crypto/drivers/hmac.c

```diff
@@ -280,7 +280,7 @@ static void msg_fifo_write(const uint8_t *message, size_t message_len) {
  *
  * @param[out] ctx Context to which values are written.
  */
-static void tmp_avoid_hw_hang(hmac_ctx_t *ctx) {
+status_t tmp_avoid_hw_hang(hmac_ctx_t *ctx) {
```

Comment:
Why the static was removed, do you plan to use this function outside this file?

Reply:
No, I don't. Thanks.